web/src/util/services/templates.ts

```diff
@@ -136,6 +136,7 @@ export async function manifestFromInstaller(i: Installer, kurlUrl: string, repli
   if (kurlVersion) {
     kurlUtilImage = `replicated/kurl-util:${kurlVersion}`;
     kurlBinUtils = `kurl-bin-utils-${kurlVersion}.tar.gz`;
+    i.spec.kurl = {additionalNoProxyAddresses: [], installerVersion: kurlVersion}
```

Comment:
```suggestion
    if (i.spec.kurl) {
      i.spec.kurl.installerVersion = kurlVersion;
    } else {
      i.spec.kurl = {additionalNoProxyAddresses: [], installerVersion: kurlVersion}
    }
```
that way you don't clobber other kurl settings here

Reply:
Good catch, I've added your suggestion. Thank you